### PR TITLE
qa: checkstyle, spotbugs

### PR DESCRIPTION
**Pull request by soloturn**
* use standard comments
* shorten line lenght
* logger was defined twice
* use {} for logger parameters
* whitespaces so no warning is produced

### CODE REVIEW

File: src/main/java/org/terasology/biomesAPI/Biome.java [hunk 1]
```diff
@@ -62,17 +50,16 @@ default float getTemperature() {
...
     default Block getSurfaceBlock(Vector3ic pos, int seaLevel) {
         BlockManager blockManager = CoreRegistry.get(BlockManager.class);
         return blockManager.getBlock("CoreAssets:Grass");
     }
 
     /**
-     * @return The block that should be generated at the given position in the biome, which isn't the surface.
-     * Defaults to dirt for the first 32 blocks and then stone below.
+     * @return The block that should be generated at the given position in the biome, which isn't the surface. Defaults to dirt for the
+     *         first 32 blocks and then stone below.
```
Comment (jdrueckert) [inline src/main/java/org/terasology/biomesAPI/Biome.java:62]: on purpose or accidental?
```suggestion
     * first 32 blocks and then stone below.
```

**Comment by jdrueckert** [comment]
Thanks for the great PR description, please correct the PR title though :wink:

File: src/main/java/org/terasology/biomesAPI/Biome.java (hunk 1 above)
Comment (soloturn) [reply src/main/java/org/terasology/biomesAPI/Biome.java:62]: on purpose. if you want to leave it feel free. just used intellij default format, shift-ctrl-alt-k and it made it so ... @jdrueckert

Comment (jdrueckert) [reply src/main/java/org/terasology/biomesAPI/Biome.java:62]: if it's on purpose and auto-indented by intellij, that's fine 👍

**Review by jdrueckert — APPROVED** [review]

**Pull request merged**